tests/orchestrator.js

```diff
@@ -0,0 +1,20 @@
+import retry from "async-retry";
+
+async function waitForAllServices() {
+  await waitForWebServer();
+
+  async function waitForWebServer() {
+    return retry(fetchStatusPage, {
+      restries: 100,
```

Comment:
Corrigir erro de digitação no nome da propriedade.

```suggestion
      retries: 100,
```

Reply:
Sugiro adicionar também a propriedade maxTimeOut, pois por padrão, o retry multiplicará o tempo entre cada tentativa por um fator de 2, ficando exponencialmente mais lento. Com o maxTimeOut, o tempo entre as tentativas nunca excederá um determinado valor, ficando assim um tempo mais próximo de linear.

```sugestion
      retries: 100,
      maxTimeOut: 1000 , // 1 segundo
```